src/mscorlib/shared/System/Reflection/ReflectionTypeLoadException.cs

```diff
@@ -40,48 +40,31 @@ public override void GetObjectData(SerializationInfo info, StreamingContext cont
             info.AddValue("Exceptions", LoaderExceptions, typeof(Exception[]));
         }
 
-        public override string Message
+        public Type[] Types { get; }
+
+        public Exception[] LoaderExceptions { get; }
+
+        public override string Message => CreateString(base.Message, LoaderExceptions, e => e.Message);
```

Comment:
These delegates and everything just produce a lot of binary bloat. I would be better for CreateString to just take a single `bool` argument that says whether we are formatting Message or string.

Reply:
Fixed